### DISCUSSION TITLE
Conserve-t-on les cuisines centrales dans notre base de données?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1163] [Question / réponse] [answered]
Dans le cas d'une cuisine centrale qui n'a pas de lieu de service sur place, conserve-t-on cette cantine dans notre base de données ou la supprime-t-on? Si on la conserve, quel en est l'intérêt puisque cette cantine ne remplira de toute façon pas de diagnostic annuel?
Ex. : cuisine centrale "La Nouvelle Cantine"

**Comment by alemangui** [comment, marked as answer]
On la conserve pour l'instant. En effet on n'a pas de fonctionnalités dédiées à ce type de cantines, mais on envisage d'en développer plus tard si le besoin se fait ressentir. Par exemple, pour une cantine centrale sans lieu de restauration on pourrait orienter le parcours vers l'ajout de ses cantines satellites, peut-être même en s'interfaçant avec leurs SI.

**Reply by ghost** [reply]
D'ac. Merci, @alemangui!
